tf_files/aws/modules/eks/init_cluster.sh

```diff
@@ -15,8 +15,9 @@ fi
 
 #KUBECTL=$(bash which kubectl)
 if ! $(bash which kubectl) --kubeconfig "${kubeconfig_path}" get daemonsets -n kube-system calico-node > /dev/null 2>&1; then
-  $(bash which kubectl) --kubeconfig "${kubeconfig_path}" apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v1.4/calico.yaml
-                                                                  # https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.1/config/v1.1/calico.yaml
+  $(bash which kubectl) --kubeconfig "${kubeconfig_path}" apply -f kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.5/config/v1.5/calico.yaml
```

Comment:
Did you put an extra  `kubectl apply -f` on the line?

Reply:
No, there is a removal and an addition according to github, which is basically a modification of the line, the one you might be seeing is the one that is commented out,